packages/docusaurus-theme-classic/src/theme/Footer/index.js

```diff
@@ -13,22 +13,22 @@ import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 import useBaseUrl from '@docusaurus/useBaseUrl';
 import styles from './styles.module.css';
 
-function FooterLink({item}) {
-  const toUrl = useBaseUrl(item.to);
+function FooterLink({to, href, label, ...props}) {
+  const toUrl = useBaseUrl(to);
   return (
     <Link
       className="footer__link-item"
-      {...item}
-      {...(item.href
+      {...props}
+      {...(href
         ? {
             target: '_blank',
             rel: 'noopener noreferrer',
-            href: item.href,
+            href,
           }
         : {
             to: toUrl,
           })}>
-      {item.label}
+      {label}
```

Comment:
Now i wonder if we should put it right before label instead ? Otherwise the props cannot override `rel` and `target` for href

Reply:
Yep, exactly, you are absolutely right, fixed!